examples/opensora_cai/opensora/models/layers/blocks.py

```diff
@@ -238,6 +317,12 @@ def construct(self, x, mask=None):
         k = ops.squeeze(k, axis=2)
         v = ops.squeeze(v, axis=2)
 
+        # WARNING: this may be a bug
```

Comment:
from my exp apply ROPE in fp32 gives a better convergence, you may have a try

Reply:
Thanks, will pay attention to this when working on the training integration.